### ISSUE TITLE
Set Default Due Date to 30 days

### ISSUE THREAD

**Issue by russdeffner** [hotosm/tasking-manager#4870]
As due date is rarely filled in, and as we push towards the goal of having projects complete in 30 days or less, we should start by setting a default of 30 days from creation.

**Comment by zlavergne** [comment]
Is that 30 day target regardless of priority? I think autofilling or suggesting to some degree would be useful but I'm thinking a blanket "30 days for any project" may give undo urgency to potentially non-urgent projects. For instance, if a PM's creates a low-pri/long term fixup project for an area.

Maybe starting at 30 days for "Urgent" and increment 30 days each priority level?

**Comment by SColchester** [comment]
Great suggestions! I'd argue good defaults might be: 
Urgent: 30 days
High: 60 days
Medium: 180 days
Low: 270 days

**Comment by russdeffner** [comment]
TL;DR: If we want to see a high number of projects getting complete, we need to push for a much shorter time-frame than what we 'typically' see.

Full text: I am not sure where the 30 day target sits at the moment (I haven't heard it talked about recently, so we can maybe think about this more 'flexibly'). However, we know imagery and mapping activity in any given area is more frequently being updated/changed so I would still argue that 'good' projects (and PMs) are created to be done in as little time as possible and if projects are made that knowingly won't be done in a month (or maybe 2-3 max); then those projects are either too big, or have too little mapper/validator attention. I would also say that although the "Project Gardening" from the Quality Assurance and Control Working Group does not have a time-frame set for the current phase, the roughly 2 years in phase 1 & 2 was not enough; projects that go beyond 6 months in my opinion are just bloating the database and will likely never get finished.

**Comment by ghimiresushma99** [comment]
Suggesting the default due dates in accordance with the priority of the project sounds reasonable. I also agree with @russdeffner that the due date should be kept to a maximum of 3-4 months. The project may be archived after the due date so that other ongoing projects can be highlighted.

Furthermore, if the project is not completed by the due date and the PMs want that, it remains active rather than archived, they may be notified via emails to update their due dates accordingly; neither the project will be archived by default.

**Comment by petya-kangalova** [comment]
Discussion in the Tasking Manager Meet Up: we should do some research on what is currently happening: number of days per urgency of projects; then we can consult with organizations. This issue requires more consultation before we move into implementation.
